Bottom: 4adb5b250ef415a0facb22ca10afd24bf18b2ff2
Top:    96dbf9775d75c3b2a0bc1e683743c605bf0c5473
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-16 14:06:10 -0500

Create certificate directories during image build



---
diff --git a/docker/docker-mediawiki/configure.sh b/docker/docker-mediawiki/configure.sh
index 06ed299..eba1977 100755
--- a/docker/docker-mediawiki/configure.sh
+++ b/docker/docker-mediawiki/configure.sh
@@ -1,6 +1,11 @@
 #!/bin/bash
 set -e
 
+# Create certificate directories
+mkdir -p /etc/grid-security/certificates
+mkdir -p /etc/ssl/certs/
+mkdir -p /etc/ssl/private
+
 # comment out apache2 config file lines that reference the environment variables
 sed -i 's|^Mutex file|#Mutex file|' /etc/apache2/apache2.conf
 sed -i 's|^PidFile |#PidFile |' /etc/apache2/apache2.conf
